library/datetime/to_time_spec.rb

```diff
@@ -1,6 +1,25 @@
 require File.expand_path('../../../spec_helper', __FILE__)
 require 'date'
 
-describe "DateTime#to_time" do
-  it "needs to be reviewed for spec completeness"
+ruby_version_is "2.4" do
+  describe "DateTime#to_time" do
+    it "yields a new Time object" do
+      DateTime.now.to_time.should be_kind_of(Time)
+    end
+
+    it "preserves the same time regardless of local time or zone" do
+      with_timezone("Pactific/Pago_Pago", -11) do
+        date = DateTime.new(2012, 12, 24, 12, 23, 00, '+03:00')
```

Comment:
Could you move this line above? So it's very clear the timezone change does not affect it :)